openshift_day2_configurator/configurators/ldap.py

```diff
@@ -440,66 +440,50 @@ def execute_ldap_configuration(
+                    "client": client,
+                },
             },
-        },
-        DISABLE_SELF_PROVISIONERS_TASK_NAME: {
-            "func": disable_self_provisioners,
-            "func_kwargs": {"client": client},
-        },
-        CREATE_LDAP_GROUPS_SYNC: {
-            "func": create_ldap_groups_sync,
-            "func_kwargs": {
-                "group_syncer_name": config.get("group_syncer_name"),
-                "group_syncer_namespace": config.get("group_syncer_namespace"),
-                "group_syncer_whitelist": config.get("group_syncer_whitelist", ""),
-                "group_syncer_schedule": config.get("group_syncer_schedule"),
-                "concurrency_policy": config.get("group_syncer_concurrency_policy"),
-                "sealed_sync_secret_name": config.get("sealed_sync_secret_name"),
-                "sealed_sync_secret_encrypted_data": config.get("sealed_sync_secret_encrypted_data"),
-                "client": client,
+            DISABLE_SELF_PROVISIONERS_TASK_NAME: {
+                "func": disable_self_provisioners,
+                "func_kwargs": {"client": client},
+            },
+            CREATE_LDAP_GROUPS_SYNC: {
+                "func": create_ldap_groups_sync,
+                "func_kwargs": {
+                    "group_syncer_name": config.get("group_syncer_name"),
+                    "group_syncer_namespace": config.get("group_syncer_namespace"),
+                    "group_syncer_whitelist": config.get("group_syncer_whitelist", ""),
+                    "group_syncer_schedule": config.get("group_syncer_schedule"),
+                    "concurrency_policy": config.get("group_syncer_concurrency_policy"),
+                    "sealed_sync_secret_name": config.get("sealed_sync_secret_name"),
+                    "sealed_sync_secret_encrypted_data": config.get("sealed_sync_secret_encrypted_data"),
+                    "client": client,
+                },
             },
         },
-    }
-
-    verify_and_execute_kwargs = {
-        "config": config,
-        "logger_obj": logger,
-        "progress": progress,
-        "logger": logger,
-    }
-
-    if progress:
-        task_id = progress.add_task(description="  Configuring LDAP", total=len(tasks_dict))
-
-    for _task, _func_config in tasks_dict.items():
-        _kwargs: Dict[str, Any] = {**verify_and_execute_kwargs, **_func_config["func_kwargs"]}  # type: ignore[dict-item]
-        status_dict.update(verify_and_execute_configurator(func=_func_config["func"], task_name=_task, **_kwargs))
-
-        if progress and task_id is not None:
-            progress.update(task_id, advance=1)
-
-    if progress and task_id is not None:
-        progress.update(task_id, advance=1)
-
-    return status_dict
+        verify_and_execute_kwargs={
+            "config": config,
+            "logger_obj": logger,
+            "progress": progress,
+            "logger": logger,
+        },
+        description="  Configuring LDAP",
```

Comment:
add the required spaces in `execute_configurator` as they are needed for all

Reply:
done